src/features/workspaces/components/WorkspaceSwitchingIndicator.tsx

```diff
@@ -41,25 +40,23 @@ const styles = theme => ({
     zIndex: 200,
   },
   spinner: {
-    width: 40,
-    height: 40,
     marginRight: 10,
   },
   message: {
     fontSize: 16,
     whiteSpace: 'nowrap',
-    color: theme.colorAppLoaderSpinner,
+    color: '#FFFFFF',
```

Comment:
What does this change exactly? And why is it needed?

Reply:
I wanted to get rid of some theme colors which are only used once, but looking at it, hardcoded white is also bad. Will undo.

Reply:
Undid the change in [`b4d3dd6` (#1410)](https://github.com/ferdium/ferdium-app/pull/1410/commits/b4d3dd6cb6ca30ff6db8a0a866339e1ba9d95683)